RadixWallet/Features/DisplayEntitiesControlledByMnemonic/DisplayEntitiesControlledByMnemonic+View.swift

```diff
@@ -93,10 +95,14 @@ extension DisplayEntitiesControlledByMnemonic {
 						)
 					}
 
-					VStack(alignment: .leading, spacing: .small3) {
-						ForEach(viewStore.accounts) { account in
-							SmallAccountCard(account: account)
-								.cornerRadius(.small1)
+					if viewStore.accounts.isEmpty {
+						NoContentView("No non-hidden accounts.\n#\(viewStore.hiddenAccounts.count) hidden accounts.") // FIXME: Strings
```

Comment:
asked for copy here: https://rdxworks.slack.com/archives/C031A0V1A1W/p1700050685956519

Reply:
Copy updated